media/css/protocol/components/_footer.scss

```diff
@@ -27,6 +27,11 @@ $image-path: '/media/protocol/img';
             &.spotify {
                 background-image: url('/media/img/logos/social/spotify-white.svg');
             }
+
+            &.mastodon {
+                background-image: url('/media/img/logos/social/mastodon-white.svg');
+                background-size: contain;
```

Comment:
Oh also I don't think this will be necessary if you use the icons I added to #12519 that are sized the same as the others.

Reply:
Haha, I _did_ use your icons! 😅 and they were larger than the rest

Reply:
I added fixed icons to the issue.